app-sso/how-to-guides/service-operators/configure-token-signature.hbs.md

```diff
@@ -123,6 +123,8 @@ You can generate an RSA key yourself using OpenSSL. Here are the steps:
 
    > More [OpenSSL key generation examples here](https://www.openssl.org/docs/man1.1.1/man1/openssl-genpkey.html).
 
+   > **Note** The minimum key size for an `Authserver` is 2048
```

Comment:
**[issue]:** Please, update the `app-sso/reference/apis/authserver` as well for `AuthServer.spec.tokenSignature.(signAndVerifyRef,extraVerifyRefs)`.